src/DeckGlMap/layers/VectorLayer.js

```diff
@@ -1,11 +1,40 @@
 import {CompositeLayer} from '@deck.gl/core';
 import {GeoJsonLayer} from '@deck.gl/layers';
-import {forIn as _forIn, includes as _includes} from 'lodash';
+import {
+	forIn as _forIn,
+	includes as _includes,
+	partition as _partition,
+} from 'lodash';
 import styleHelpers from '../helpers/style';
+import constants from '../../constants';
 
 class VectorLayer extends CompositeLayer {
+	constructor(props) {
```

Comment:
This is not necessary if you don't declare any property.

Reply:
https://github.com/gisat-panther/ptr-maps/pull/116/commits/257b71898db9b9366ccf0c41dd5007b504abd525